devtools/gradle/src/main/java/io/quarkus/gradle/AppModelGradleResolver.java

```diff
@@ -273,6 +273,13 @@ private void collectDependencies(Configuration config, AppModel.Builder appBuild
                             paths.add(resourcesDir.toPath());
                         }
                     }
+
+                    for (File outputDir : depProject.getTasks().findByName(JavaPlugin.PROCESS_RESOURCES_TASK_NAME)
+                            .getOutputs().getFiles()) {
+                        if (outputDir.exists()) {
+                            paths.add(outputDir.toPath());
+                        }
+                    }
```

Comment:
This change should now move to https://github.com/quarkusio/quarkus/blob/master/devtools/gradle/src/main/java/io/quarkus/gradle/builder/QuarkusModelBuilder.java#L285

Reply:
Okk